cvmfs/cvmfs_config

```diff
@@ -1117,6 +1118,28 @@ cvmfs_bugreport() {
    echo "JIRA bug report to https://sft.its.cern.ch/jira/browse/CVM"
 }
 
+cvmfs_fsck() {
```

Comment:
In order to avoid the name clash with /usr/bin/cvmfs_fsck, this might be better called `cvmfs_fsck_all`